cypress/e2e/onboarding.spec.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+
+describe('Onboarding Page', () => {
+  afterEach(() => {
+    cy.truncateDatabase();
+  });
```

Comment:
is this necessary to do for onboarding related tests?

Reply:
I'd argue it doesn't hurt to clean up... get rid of any users created so there's a fresh slate for whatever tests may run after this file. Granted, it could be an `after()`...